ee/clickhouse/queries/test/__snapshots__/test_retention.ambr

```diff
@@ -656,6 +656,191 @@
            intervals_from_base
   '
 ---
+# name: TestClickhouseRetention.test_groups_filtering_person_on_events_v2
+  '
+  
+  SELECT distinct_id,
+         person_id
+  FROM events
+  WHERE distinct_id IN ('person1',
+                        'person2')
+  GROUP BY distinct_id,
+           person_id
+  ORDER BY if(distinct_id = 'person1', -1, 0)
+  '
+---
+# name: TestClickhouseRetention.test_groups_filtering_person_on_events_v2.1
+  '
+  WITH actor_query AS
+    (WITH 'Day' as period,
+          NULL as breakdown_values_filter,
+          NULL as selected_interval,
+          returning_event_query as
+       (SELECT toStartOfDay(toTimeZone(toDateTime(e.timestamp, 'UTC'), 'UTC')) AS event_date,
+               if(notEmpty(overrides.person_id), overrides.person_id, e.person_id) as target
+        FROM events e
+        LEFT OUTER JOIN
```

Comment:
and this

Reply:
Q: Why did we have to do the whole person override join here? Seems like it's a groups query, which just depends on events, not person IDs?